testsuite/mockserver.py

```diff
@@ -13,30 +13,25 @@ class Mockserver:
     def __init__(self, url):
         self.url = url
 
-    def create_expectation(self, expectation_id, path, opa_policy,
-                           content_type: Union[ContentType, str] = ContentType.PLAIN_TEXT):
+    def create_expectation(
+        self, expectation_id, path, opa_policy, content_type: Union[ContentType, str] = ContentType.PLAIN_TEXT
+    ):
         """Creates an Expectation - response with body that contains OPA policy (Rego query)"""
         response = httpx.put(
-            urljoin(self.url, "/mockserver/expectation"), verify=False, timeout=5, json={
-                    "id": expectation_id,
-                    "httpRequest": {
-                        "path": path
-                    },
-                    "httpResponse": {
-                        "headers": {
-                            "Content-Type": [str(content_type)]
-                        },
-                        "body": opa_policy
-                    }
-                }
-            )
+            urljoin(self.url, "/mockserver/expectation"),
+            verify=False,
+            timeout=5,
+            json={
```

Comment:
JSON one-line formatting is arguably worse readable.

Reply:
There is an option to leave dictionary formatting in the "traditional way" if black sees a trailing comma after the last element.

Reply:
That would be great, but black can't be configured if I remember correctly. Its rules are set (that is what is good about black). But we can still use `ignore`

Reply:
@averevki You are right! Thanks for the suggestions. https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma. It is not a config option, but if you have a comma after the last element in dict it won't get squashed to a oneliner. @jsmolar Could you show me all locations where you would like to get rid of one-line dicts?